Makefile

```diff
@@ -33,26 +31,16 @@ fmt:
 .PHONY: clean
 clean:
 	rm doc.go
-	rm -f $(GOPATH)/bin/$(PROGNAME)
 	$(GO) clean $(GOFLAGS) -i $(PKG)
 
-.PHONY: cross
-cross:
-	gox $(GOVERSIONLDFLAG) \
-		-output 'build/{{.Dir}}/{{.OS}}_{{.Arch}}/biscuit' \
-		-os "linux darwin windows" \
-		-arch "amd64 arm arm64 386" \
-		-osarch '!darwin/arm !darwin/386 !darwin/arm64'
-	./cross.sh
-
-.PHONY: docker-build
-docker-build:
-	docker build -f Dockerfile.e2e -t $(PROGNAME)/local .
-
-.PHONY: docker-cross
-docker-cross: docker-build
-	mkdir build_docker_cross || /bin/true
-	docker run -v $(shell pwd)/build_docker_cross/:/tmp/build/ $(PROGNAME)/local /bin/bash -xe -c \
-		"rm -f build && ln -s /tmp/build/ build && make cross"
```

Comment:
We'll need to fix the tests to run with the newly build docker image (which is built on alpine, so won't even have bash).